cypress/tests/api/api-bankaccounts.spec.ts

```diff
@@ -16,6 +16,11 @@ type TestBankAccountsCtx = {
 describe("Bank Accounts API", function () {
   let ctx: TestBankAccountsCtx = {};
 
+  before(() => {
```

Comment:
Why did this need to be added? Why does the dev server change our tests, that seems weird.

Reply:
For some reason when the api was visited first, the e2e tests couldn't connect to the front end web app and execute the tests. Adding a visit to the front end here first made it so they could both execute in the same run. Super weird but did seem related to the vite upgrade.

Reply:
Odd 🤔